framework/src/main/java/fi/otavanopisto/pyramus/koski/StudentSubjectSelections.java

```diff
@@ -3,11 +3,23 @@
 import org.apache.commons.lang3.ArrayUtils;
 import org.apache.commons.lang3.StringUtils;
 
+import fi.otavanopisto.pyramus.koski.koodisto.KoskiOppiaineetYleissivistava;
+
 public class StudentSubjectSelections {
 
   public StudentSubjectSelections() {
   }
 
+  public KoskiOppiaineetYleissivistava koskiKoodi(String languageCode) {
+    return
+        isALanguage(languageCode) ? KoskiOppiaineetYleissivistava.A1 :
```

Comment:
A and A1 both map to KoskiOppiaineetYleissivistava.A1?

Reply:
Yes - the mapping is weird and was also changed for 2019.